versioned_hdf5/subchunk_map.py

```diff
@@ -17,6 +17,13 @@
 )
 from numpy.typing import NDArray
 
+# Temporary hack to work around pytest issue: if a pure-pyton Cython module is first
+# imported by `pytest .` and it contains doctests, it will be imported in pure python
+# instead of its compiled form. This fails in CI, as Cython is not a runtime dependency,
+# and regardless would cause a lot of type-checking normally performed by Cython to be
+# skipped so it is to be avoided.
+from . import hyperspace  # noqa: F401
```

Comment:
This will later be replaced by a proper import of functionality

Reply:
Understood. Maybe the later revision will address this, but 

1. we can also make `cython` a test dependency if needed, and
2. I don't think we have any doctests?

Reply:
hyperspace.py has two doctests.
Making cython a test dependency would be a bad idea because, while it would fix the CI failure, it would cause ALL tests to use the pure-python version of hyperspace.py, which is much more lenient in terms of typing than the cythonized version, thus potentially hiding breaking bugs completely until someone manually tries using it.